orbit/pkg/packaging/windows.go

```diff
@@ -149,19 +149,59 @@ func BuildMSI(opt Options) (string, error) {
 		}
 	}
 
-	if err := wix.Heat(tmpDir, opt.NativeTooling, opt.LocalWixDir); err != nil {
+	absWixDir := opt.LocalWixDir
+	wineChecked := false
+
+	// Download wix for macOS running on arm64, unless a local-wix-dir is provided
+	if runtime.GOOS == "darwin" && runtime.GOARCH == "arm64" && absWixDir == "" {
+		const wixDownload = "https://github.com/wixtoolset/wix3/releases/download/wix3112rtm/wix311-binaries.zip"
+		fmt.Println("Detected macOS arm64. fleetctl must use locally installed wine and wix to build the MSI package.")
+
+		// Ensure wine is installed before downloading wix
+		if err = checkWine(false); err != nil {
+			return "", err
+		}
+		wineChecked = true
+
+		fmt.Printf("Downloading wix from %s\n", wixDownload)
+		wixZip := filepath.Join(tmpDir, "wix.zip")
+		cmd := exec.Command("curl", "--location", "--output", wixZip, wixDownload)
```

Comment:
Why is curl a dependency? Shouldn't we just use Go to download the WiX zip?

Reply:
ok